sound/soc/intel/boards/bdw-rt5677.c

```diff
@@ -342,7 +350,7 @@ static struct snd_soc_dai_link bdw_rt5677_dais[] = {
 		.dpcm_playback = 1,
 		.dpcm_capture = 1,
 		.init = bdw_rt5677_init,
-		SND_SOC_DAILINK_REG(dummy, be, dummy),
+		SND_SOC_DAILINK_REG(ssp0_port, be, platform),
```

Comment:
@plbossart SOF can use dummy too its just that we cannot use the same dummy component for both the platform and the CPU DAIs. they have to be different

Reply:
why would we use a dummy CPU dai when we explicitly expose one in bdw.c?

Reply:
hmm good question but the ssp0-port CPU DAI has no special properties that the dummy doesnt have. Also, if we use dummy (different dummies), both the legacy and SOF drivers can use the same definitions.

Reply:
why do we declare this in bdw.c then?

````c
/* Broadwell DAIs */
static struct snd_soc_dai_driver bdw_dai[] = {
{
	.name = "ssp0-port",
},
{
	.name = "ssp1-port",
},
};
````

Reply:
IMO it is not needed

Reply:
I think it's more explicit to do so. we do use SSP0 so it's better to make it clear.